src/test/java/uk/gov/hmcts/reform/sendletter/controllers/SendLetterControllerTest.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.hmcts.reform.sendletter.controllers;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+import uk.gov.hmcts.reform.authorisation.exceptions.InvalidTokenException;
+import uk.gov.hmcts.reform.authorisation.validators.AuthTokenValidator;
+import uk.gov.hmcts.reform.sendletter.services.LetterService;
+
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Matchers.anyString;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@RunWith(SpringRunner.class)
+@WebMvcTest
+public class SendLetterControllerTest {
+
+    @Autowired private MockMvc mockMvc;
+    @MockBean private LetterService letterService; //NOPMD
```

Comment:
pmd complains about not used variable, but we do need a mock here, hence suppression comment